db/app.js

```diff
@@ -1,12 +1,13 @@
 const express = require("express");
 const app = express();
 
-const { getTopics, getArticles, getArticleById } = require("./controllers");
+const { getTopics, getArticles, getArticleById, getCommentsByArticleId } = require("./controllers");
 app.use(express.json());
```

Comment:
This is here a bit early .... shouldn't really be needed until you patch/post something. Following screenshots leads to misunderstanding of code, like the fact that this is here early. Make sure you understand what you are writing.